src/components/Trade/Trade.tsx

```diff
@@ -42,8 +32,30 @@ export const Trade = ({ defaultBuyAssetId }: TradeProps) => {
       const result = await getDefaultAssets()
       if (!result) return
       const { buyAsset, sellAsset } = result
-      methods.setValue('sellTradeAsset.asset', sellAsset)
-      methods.setValue('buyTradeAsset.asset', buyAsset)
+      const swapperState: Partial<SwapperState> = {
```

Comment:
Since we're zero-ing out everything, what about not making this a partial and also adding `slippage` (`DEFAULT_SLIPPAGE`) and `isSendMax` (false) ?